mandatory/3-function-output.js

```diff
@@ -5,12 +5,13 @@ function getRandomNumber() {
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
-  return word1.concat(word2);
+  return word1.concat("",word2);
```

Comment:
The concat change here with the double quotes "" is not necessary. Only thing required in this part is adding a comment line above combine2Words